addons/use.md

```diff
@@ -1,19 +1,23 @@
-# Use Addon
+---
+title: 使用附带组件
```

Comment:
```suggestion
title: 使用扩展插件
```

WDYT?

Reply:
> WDYT?

Now, the website is using `附带组件` instead of `拓展插件`. So I didn't change it. 

![](https://user-images.githubusercontent.com/83524927/180386778-5a262fca-08d1-4132-b824-8b41cfb1b970.jpeg)

Reply:
I don't have strong opinion, I'll let @QC-L or @Ivocin do the decision

Reply:
> I don't have strong opinion, I'll let @QC-L or @Ivocin do the decision

OK & thanks

Reply:
感觉扩展插件更合适，麻烦统一调整一下，多谢

Reply:
> 感觉扩展插件更合适，麻烦统一调整一下，多谢

已修改（ 9407543 ）。